### PR TITLE
Example vite and webpack5

### PR DESCRIPTION
**Pull request by KonghaYao**
After updating to version 7.0.0, I created a simple example for both dev and production modes using Vite and Webpack 5. The example includes some adaptation code for Vite and Webpack, which I believe will be helpful for those who are not very familiar with web workers.

### CODE REVIEW

File: examples/webpack5/src/vite-env.d.ts
Comment (josdejong) [inline examples/webpack5/src/vite-env.d.ts:1]: Can you remove this vite file from the webpack5 example? I think it would be confusing :)

**Review by josdejong — COMMENTED** [review]
Thanks, this is great and indeed very helpful I think!

File: examples/webpack5/src/vite-env.d.ts
Comment (KonghaYao) [reply examples/webpack5/src/vite-env.d.ts:1]: Oops! I forgot to delete this file! It has been modified now. 😂

Comment (josdejong) [reply examples/webpack5/src/vite-env.d.ts:1]: 💪

**Comment by KonghaYao** [comment]
You're welcome. Workerpool has been very helpful in my project, and I also want to help it get better and better.

**Comment by josdejong** [comment]
Thanks, appreciated!

**Pull request merged**